.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "all"
+}
```

Comment:
This project already has a _.editorconfig_ file. Prettier already reads options from there. If they’re specified here as well, this might cause conflicts.

I suggest to add the following line to _.editorconfig_ (based on this config):

```ini
max_line_length = 80
```

Then reduce this file to:

```json
{
  "singleQuote": true,
  "trailingComma": "all"
}
```

I also suggest renaming this file to `prettierc.json`, so it works better with other tooling, including prettier (which uses globs).

Reply:
I agree with the fact that the .editorconfig and prettier config should be more aligned, so they don't conflict.

> Prettier already reads options from there.

I am not sure if this is true or not. Documentation of prettier is not very conclusive about this. Only place where there .editorconfig is mentioned is here: https://prettier.io/docs/en/api.html#prettierresolveconfigfilepath-options

And then it says "If options.editorconfig is true and an .editorconfig file is in your project, Prettier will parse it and convert its properties to the corresponding prettier configuration. This configuration will be overridden by .prettierrc, etc."

There is no documentation to be found on what the default value for that option is.

Reply:
File renamed, and added / removed some options.